### PR TITLE
Ehienabulele doc patch

### PR DESCRIPTION
**Pull request by ehienabulele**
Added the doc for "Hibernating your Cluster" and "Hibernating Namespaces with Tenant Operator"

### CODE REVIEW

**Comment by stakater-user** [comment]
@ehienabulele Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/StakaterCloudDocs/commit/a8cb4b7d94c81fda4e031e8f663eeb0c0235abe1/checks) has Failed!

**Comment by stakater-user** [comment]
@ehienabulele Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/StakaterCloudDocs/commit/df64b00186f6411223a849353c340ba09634e48b/checks) has Failed!

**Comment by stakater-user** [comment]
@ehienabulele Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/StakaterCloudDocs/commit/6ddaba79165c535fd8c0045324386af042cd56b0/checks) has Failed!

**Comment by stakater-user** [comment]
@ehienabulele Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/StakaterCloudDocs/commit/8821f8365b5d149e917f8d772ad1707a072c8d49/checks) has Failed!

**Comment by stakater-user** [comment]
@ehienabulele Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/StakaterCloudDocs/commit/44cf4e2bd88b1a821f1f740d97ec0c435ab842a9/checks) has Failed!

File: docs/.vuepress/configs/sidebar.ts [hunk 1]
```diff
@@ -29,6 +29,12 @@ export const sidebarEn: SidebarConfig = [
       "/content/sre/cloud-provider/binero.md",
     ],
   },
+  {
+    text: "Cluster Management",
+    collapsible: true,
+     children: [
+       "/content/sre/cluster-management/imgs/hibernating-your-cluster.md"],
```
Comment (abdulhaseeb2) [inline docs/.vuepress/configs/sidebar.ts]: please update this to 
`
"/content/sre/cluster-management/imgs/hibernating-your-cluster.md",
],
`

**Review by abdulhaseeb2 — CHANGES_REQUESTED** [review]

**Comment by stakater-user** [comment]
@ehienabulele Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/StakaterCloudDocs/commit/35d8a668f072c9661c1c9dced55b7f7571b3785e/checks) has Failed!

**Comment by stakater-user** [comment]
@ehienabulele Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/StakaterCloudDocs/commit/5c621672ca2e49d2ebf420f0e13a09c941dc340e/checks) has Failed!

**Comment by stakater-user** [comment]
@ehienabulele Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/StakaterCloudDocs/commit/1de3f142d07fb85abc9c79664bf47f5d2196408f/checks) has Failed!

File: docs/.vuepress/configs/sidebar.ts [hunk 2]
```diff
@@ -29,6 +29,12 @@ export const sidebarEn: SidebarConfig = [
       "/content/sre/cloud-provider/binero.md",
     ],
   },
+  {
+    text: "Cluster Management",
+    collapsible: true,
+     children: [
+       "/content/sre/cluster-management/imgs/hibernating-your-cluster.md",],
```
Comment (abdulhaseeb2) [inline docs/.vuepress/configs/sidebar.ts]: This path is wrong, it should be `/content/sre/cluster-management/hibernating-your-cluster.md`

**Review by abdulhaseeb2 — CHANGES_REQUESTED** [review]

**Comment by stakater-user** [comment]
@ehienabulele Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/StakaterCloudDocs/commit/895d56ce996975fbc363f53cd66379c6bd0340ba/checks) has Failed!

**Comment by stakater-user** [comment]
@ehienabulele Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/StakaterCloudDocs/commit/8cfbae12debc11cb0a157b15a24505369b5478cd/checks) has Failed!

**Comment by stakater-user** [comment]
@abdulhaseeb2 Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/StakaterCloudDocs/commit/24ff1d13bb2ee187e00fe7fe9e13b3dc85c032e4/checks) has Failed!

**Comment by stakater-user** [comment]
@ehienabulele Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/StakaterCloudDocs/commit/e2b65a7bcbd5f3130d399caa182831cc01adfd2c/checks) has Failed!

**Comment by stakater-user** [comment]
@hussnain612 Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-36-c6f25af0`

**Comment by stakater-user** [comment]
@ehienabulele Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-36-8862ee60`

**Comment by stakater-user** [comment]
@ehienabulele Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-36-e35f2cc3`

**Comment by stakater-user** [comment]
@ehienabulele Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-36-aef87379`

**Comment by stakater-user** [comment]
@ehienabulele Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-36-9acf2cf2`

**Review by callum-stakater — APPROVED** [review]

**Comment by stakater-user** [comment]
@callum-stakater Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:v0.0.21`

**Pull request merged**